docker/dev/tls/docker-compose.core-override.example.yml

```diff
@@ -13,6 +13,19 @@ services:
       # It must be amended accordingly to OS.
       - /etc/ssl/certs/ca-certificates.crt:/etc/ssl/certs/ca-certificates.crt:ro
 
+  backend-test:
+    # Connect the backend-test container to the same network as the backend for nextcloud HTTP interactions
+    networks:
+      - external
+    extra_hosts:
+      - "nextcloud.local:host-gateway"
```

Comment:
usually this should be nedded, as long as the `backend-test` service is in the external network. With the proposed setup, the `traefik` service should have the alias `nextcloud.local`.

Reply:
Thanks! Will update 👍🏾